src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -155,6 +155,8 @@ public static void main(String[] args) throws Exception {
         options.addOption(null, "loaderthreads", true, "Number of loader threads (default 10)");
         options.addOption(null, "enableforeignkeys", true, "Whether to enable foregin keys");
 
+        options.addOption(null, "warmuptime", true, "Warmup time in seconds for the benchmark");
+        options.addOption(null, "initialdelay", true, "Delay in seconds for starting the benchmark");
```

Comment:
Prefer having the unit in these metrics. So `warmup_time_secs` and `initial_delay_secs` so this is clearer.

Reply:
i will add secs. But none of the other variables have '_' in them. SO will rename to warmuptimesecs and initialdelaysecs.